[Content Patcher] Update Chinese documentation

SummerFleur2997 opened the pull request:
1. Updated multiple instances of inappropriate or overly colloquial expressions in the Chinese documentation, revising them with more formal language to match the style of technical documentation.
2. Optimized the document formatting and corrected improper punctuation usage based on Chinese punctuation standards.
3. Due to some force majeure factors, most of the top editors of the Stardew Chinese wiki have stopped updating content on the official wiki. Therefore, the original links have been replaced with those of the currently more active Chinese community wiki(hereinafter referred to as bwiki). The content in this wiki is also published under the CC-BY-NC-SA 3.0 license. As one of the currently active editors in the bwiki, we will continue to improve the content in the Modding section.

Several leading editors in our bwiki are also very keen to further improve the Chinese localization of the CP documentation and synchronize it to bwiki, bring a more convenient experience for Chinese mod developers. If you have any questions or suggestions regarding my localization work, please feel free to let us know. We would be glad to assist in further refining the localization together.

SummerFleur2997 committed:
Refactor Chinese documentation for clarity and consistency

SummerFleur2997 committed:
fix invalid syntax

Pathoschild commented:
Thanks for all the work! I'll review the changes in-depth soon.

However, I'm very curious about the wiki situation. if you're willing, could you message me separately (e.g. on Discord) to discuss what happened, and what the issues are with the official wiki?

SummerFleur2997 commented:
> Thanks for all the work! I'll review the changes in-depth soon.
> 
> However, I'm very curious about the wiki situation. if you're willing, could you message me separately (e.g. on Discord) to discuss what happened, and what the issues are with the official wiki?

Sure, I'll reach out to you later when I’m available. 😉

Mushymato commented on ContentPatcher/docs/zh/README.md:
```diff
@@ -1,68 +1,66 @@
   <td>
 
-默认`false`（否）。是否启用[专为内容包模组作者设计的调试功能](author-guide/troubleshooting.md#debug-mode)。
+默认为 `false`。是否启用专为内容包模组作者设计的[调试功能](author-guide/troubleshooting.md#debug-mode)。
```
Should keep （否） for clarification

SummerFleur2997 replied:
```suggestion
默认为 `false`（否）。是否启用专为内容包模组作者设计的[调试功能](author-guide/troubleshooting.md#debug-mode)。
```

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -75,67 +74,64 @@ Content Patcher 可让你只使用 JSON 文件更改游戏内容。JSON 是一
 
-### 什么是数据素材？<a name="what-is-an-asset"></a>
- _素材(assets)_ 是指游戏从`Content`内容文件夹（或者从其他模组文件夹）中加载的图像、数据模型或地图。这些都是 Content Patcher 可以更改的内容。
+### 什么是素材？
```
Need to keep `<a name="what-is-an-asset"></a>` anchor link for ToC

SummerFleur2997 replied:
```suggestion
### 什么是素材？<a name="what-is-an-asset"></a>
```

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -75,67 +74,64 @@ Content Patcher 可让你只使用 JSON 文件更改游戏内容。JSON 是一
+`Format` 字段是您设计内容包所使用的 Content Patcher 版本。此字段用于保持内容包与未来版本的兼容性。
 
-你应该使用最新的格式版本（现在是`2.5.0`）来启用最新功能，避免某些被弃用的代码，减少加载模组的时间。
+您应该始终使用最新格式版本（当前为 `2.5.0`）以启用最新功能、避免使用被弃用的代码、减少加载模组的时间。
```
This was missed in original TL, current latest version is 2.7.0

SummerFleur2997 replied:
```suggestion
您应该始终使用最新格式版本（当前为 `2.7.0`）以启用最新功能、避免使用被弃用的代码、减少加载模组的时间。
```

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -224,29 +217,29 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
          "Target": "Maps/springobjects",
          "FromFile": "assets/fish-object.png",
-         "ToArea": { "X": 160, "Y": 80, "Width": 16, "Height": 16 }
+         "ToArea": { "X": 160, "Y": 80, "Width": 16, "Height": 16 } // 替换金枪鱼精灵图
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -255,71 +248,69 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
             "Target": "Maps/Town",
             "FromFile": "assets/town.tmx",
-            "ToArea": { "X": 22, "Y": 61, "Width": 16, "Height": 13 }
+            "ToArea": { "X": 22, "Y": 61, "Width": 16, "Height": 13 } // 替换鹈鹕镇广场
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -255,71 +248,69 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
 <td>
 
-`Include`从另外一个JSON文件里加载更多补丁。这只是将内容包组织到子文件的方式，而不是将所有内容都放在一个`content.json`中。被引用的补丁和`content.json`里的补丁功能一致。
+`Include` 从另一个 JSON 文件中加载补丁。这只是将内容包组织成多个文件的一种方式，而不是将所有内容都放在一个 `content.json` 中。包含的补丁就像直接在 `content.json` 中一样工作。
```
nit: wording
`包含的补丁就像直接在 `content.json` 中一样工作。`
should perhaps be
`包含的补丁和直接放在 `content.json` 中的补丁功能一致。`

the key aspect of this line is that you should write patches in includes identical to how they would be written as part of content.json directly

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -255,71 +248,69 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
         {
             "Action": "Include",
-            "FromFile": "assets/john_{{season}}.json"
+            "FromFile": "assets/john_{{season}}.json" // 根据季节加载不同的文件
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -255,71 +248,69 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
             "Action": "Load",
             "Target": "Portraits/Abigail",
-            "FromFile": "assets/abigail-{{season}}.png"
+            "FromFile": "assets/abigail-{{season}}.png" // 根据季节加载不同肖像
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -329,21 +320,19 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
             "FromFile": "assets/abigail-married.png",
             "When": {
-                "Spouse": "Abigail"
+                "Spouse": "Abigail" // 如果玩家她结婚，加载结婚后的肖像
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -358,21 +347,19 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
             "FromFile": "assets/john.json",
             "When": {
-                "EnableJohn": true
+                "EnableJohn": true // 当 EnableJohn 为真时加载 john.json
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -381,20 +368,19 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
             "Target": "Characters/Dialogue/MarriageDialogueAbigail",
             "Entries": {
-                "Rainy_Day_4": "{{i18n: rainy-day}}"
+                "Rainy_Day_4": "{{i18n: rainy-day}}" // 使用翻译后的文本
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -403,125 +389,119 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
 
 ### 触发动作<a name="trigger-actions"></a>
-Content Patcher 为特定使用场景添加自定义[触发动作](https://zh.stardewvalleywiki.com/模组:触发动作)，例如更新已有存档内容中的内置ID。
+Content Patcher 为特殊情况添加了自定义 [触发操作](https://wiki.biligame.com/stardewvalley/模组:触发动作)，例如更新重命名内容 ID 的旧存档。
```
nit: wording
original en doc describes scenario of updating the (internal) ids that have been stored to an existing save, e.g. an object's id changing from `Thing` to `ModId_Thing`. New translation feels more ambiguous.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -403,125 +389,119 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
     "Target": "Maps/Town",
     "SetProperties": {
-        "CurrentTime": "{{Time}}"
+        "CurrentTime": "{{Time}}" // 设置当前时间
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -403,125 +389,119 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
    "FromFile": "assets/cursors.de.png",
    "When": {
-      "Language": "de"
+      "Language": "de" // 仅在德语时应用此补丁
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -403,125 +389,119 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
+   "FromFile": "assets/cursors.{{language}}.png", // 使用 {{language}} 标记自动选择语言版本
    "When": {
-      "HasFile:{{FromFile}}": true
+    "HasFile:{{FromFile}}": true // 确认文件存在
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide.md:
```diff
@@ -403,125 +389,119 @@ You can [unpack the game's content files](https://zh.stardewvalleywiki.com/模
    "FromFile": "assets/cursors.png",
    "When": {
-      "HasFile: assets/cursors.{{language}}.png": false
+      "HasFile: assets/cursors.{{language}}.png": false // 确认没有翻译版本
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide/action-editdata.md:
```diff
@@ -225,41 +213,40 @@
   <td>
 
- _（可选）_ 素材名称中要匹配的地区代码，比如设置`"TargetLocale": "fr-FR"`只编辑法语形式的素材（比如`Data/Achievements.fr-FR`）。可以为空，只有只编辑没有地域区分的基本素材。
+（可选）素材名称中要匹配的地区代码，例如设置 `"TargetLocale": "fr-FR"` 将会只编辑法语的素材（例如 `Data/Achievements.fr-FR`）。可以为空，只有只编辑没有地域区分的基本素材。
```
mistake in existing tl:
`只有只编辑没有地域区分的基本素材。`
to
`只编辑没有地域区分的基本素材。`

SummerFleur2997 replied:
```suggestion
（可选）素材名称中要匹配的地区代码，例如设置 `"TargetLocale": "fr-FR"` 将会只编辑法语的素材（例如 `Data/Achievements.fr-FR`）。可以为空，若为空则将只编辑没有地区代码的基本素材。
```

Mushymato commented on ContentPatcher/docs/zh/author-guide/action-editimage.md:
```diff
@@ -1,43 +1,43 @@
-`Target`  | 需编辑的[游戏素材名](../author-guide.md#what-is-an-asset)（或多个由逗号分隔的素材名），比如`Portraits/Abigail`。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
-`FromFile` | 内容包文件夹中要修补到目标中的图像的相对路径（例如`assets/dinosaur.png`），或多个逗号分隔的路径。这可以是`.png`或`.xnb`文件。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
+`Action`  | 要进行的更改类型。此操作类型设置为 `EditImage`。
+`Target`  | 要替换的目标[游戏素材名称](../author-guide.md#what-is-an-asset)（或多个由逗号分隔的素材名），例如 `Portraits/Abigail`。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
```
nit: wording
EditImage should be edit and not replace, though this mistake is also in the eng docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide/action-editmap.md:
```diff
@@ -265,15 +267,15 @@ _（可选）_ 何将 `FromArea` 应用于 `ToArea`。默认为 `ReplaceByLayer`
+`TextOperations` 字段可以编辑一个现有地图属性的值（请参阅[文本操作](../author-guide.md#text-operations)）。
 
-此处`Target`只允许`["MapProperties", "PropertyName"]`，`PropertyName`为需要编辑的地图属性。
+有效的地图属性格式只有 `["MapProperties", "PropertyName"]`，其中 PropertyName 是要更改的地图属性名称。
```
the "path" mentioned in original en wording is the `Target` field in `TextOperations`, not EditMap's `Target`

Mushymato commented on ContentPatcher/docs/zh/author-guide/action-load.md:
```diff
@@ -54,38 +54,35 @@
 
-* `Low`（低），`Medium`（中），或 `High`（高）：只有最高优先级的补丁会生效。如果多个补丁的优先级一致，列表里靠前的补丁生效。
-* `Exclusive`（独占）：如果某一个补丁使用此优先级，那其他所有补丁都无效。如果多个补丁使用此优先级， **所有补丁都不生效** ，并且显示错误信息。
+* `Low`、`Medium` 或 `High`：只有最高优先级的补丁会被应用。如果多个补丁的优先级一致，则按加载顺序 + 补丁顺序列表中靠前的补丁生效。
```
nit: wording
Should keep the zh clarification on meaning of the priority values (低，中，高，独占）

Mushymato commented on ContentPatcher/docs/zh/author-guide/config.md:
```diff
@@ -128,31 +128,31 @@
-    "config.Material.description": "The material style for the billboard background.",
-    "config.Material.values.Wood": "wood",
-    "config.Material.values.Metal": "metal"
+    "config.Material.name": "Material",         // 为名为“Material”的选项添加翻译键
```
comment not in original en docs.

Mushymato commented on ContentPatcher/docs/zh/author-guide/tokens.md:
```diff
@@ -566,7 +566,7 @@ summary`](../author-guide.md#patch-summary)查看地点的内部ID。
 
 按出生顺序排列，可用[`valueAt` 参数](#valueAt)指定。比如 
-`{{ChildNames |valueAt=0}}` 和 `{{ChildGenders |valueAt=0}}` 分别是你最年长的孩子的名字和性别。
+`{{ChildNames |valueAt=0}}` 和 `{{ChildGenders |valueAt=0}}` 分别是您最年长的孩子的名字和性别。
```
nit: wording
perhaps makes more sense to say "the player's" or omit.
en wording here doesn't mention any subject

Mushymato commented on ContentPatcher/docs/zh/author-guide/translations.md:
```diff
@@ -122,27 +120,26 @@ _其他_
 
-假设你直接用`{{i18n: today}}`那文本会显示为`"今天是{{DayOfWeek}}"`；
-你可以像这样把令牌传入：`{{i18n: today |dayOfWeek={{DayOfWeek}} }}`，这样做以后文本会显示为`"今天是Monday"`。
+假设您直接使用 `{{i18n: today}}`，那么游戏内将显示为 `"今天是{{DayOfWeek}}"`。您可以像这样传递令牌：`{{i18n: today |dayOfWeek={{DayOfWeek}} }}`，这样做以后，文本才能正确地显示为 `"今天是星期一"`。
```
DayOfWeek令牌不会被翻译所以这里应该是`"今天是Monday"`

SummerFleur2997 replied:
```suggestion
假设您直接使用 `{{i18n: today}}`，那么游戏内将显示为 `"今天是{{DayOfWeek}}"`。您可以像这样传递令牌：`{{i18n: today |dayOfWeek={{DayOfWeek}} }}`，这样做以后，文本才能正确地显示为 `"今天是Monday"`。
```

Mushymato commented on ContentPatcher/docs/zh/conditions-api.md:
```diff
@@ -182,49 +179,48 @@ var conditions = api.ParseConditions(
 
 
-条件API在`GameLaunched`两tick（更新）后可使用。这和Content Patcher的生命周期有关：
+条件 API 需要在 `GameLaunched` 事件发生后两帧（Tick）起才可使用。这和 Content Patcher 的生命周期有关：
```
Not sure if a game tick should be translated as 帧 (usually meaning frame) especially in context of GameLaunched.

Mushymato commented on ContentPatcher/docs/zh/extensibility.md:
```diff
@@ -295,4 +292,4 @@ api.RegisterToken(this.ModManifest, "Initials", new InitialsToken());
 
 ## 参见<a name="see-also"></a>
-* 其他信息详见[README](README.md)
+* 其他操作和选项请参见[模组作者指南](../author-guide.md)。
```
original en docs link to README, since this is a top level page.

SummerFleur2997 replied:
```suggestion
* 其他操作和选项请参见[README](README.md)。
```

Mushymato commented on ContentPatcher/docs/zh/conditions-api.md:
```diff
@@ -182,49 +179,48 @@ var conditions = api.ParseConditions(
 
 ## 参见<a name="see-also"></a>
-* 其他信息详见[README](README.md)
+* 其他操作和选项请参见[模组作者指南](../author-guide.md)。
```
original en docs link to README, since this is a top level page.

SummerFleur2997 replied:
```suggestion
其他操作和选项请参见[README](README.md)。
```

Mushymato commented on ContentPatcher/docs/zh/token-strings-api.md:
```diff
@@ -145,48 +144,47 @@ var tokenString = api.ParseTokenString(
 
 ## 参见<a name="see-also"></a>
-* 其他信息详见[README](README.md)
+* 其他操作和选项请参见[模组作者指南](../author-guide.md)。
```
original en docs link to README, since this is a top level page.

SummerFleur2997 replied:
```suggestion
* 其他操作和选项请参见[README](README.md)。
```

Mushymato commented on ContentPatcher/docs/zh/token-strings-api.md:
```diff
@@ -145,48 +144,47 @@ var tokenString = api.ParseTokenString(
 <dd>
 
-令牌字符串API在`GameLaunched`两tick（更新）后可使用。这和Content Patcher的生命周期有关：
+令牌字符串 API 需要在 `GameLaunched` 事件发生后两帧（Tick）起才可使用。这和 Content Patcher 的生命周期有关：
```
Not sure if a game tick should be translated as 帧 (usually meaning frame) especially in context of GameLaunched.

SummerFleur2997 commented on ContentPatcher/docs/zh/author-guide/action-load.md:
```diff
@@ -54,38 +54,35 @@
-* `Low`（低），`Medium`（中），或 `High`（高）：只有最高优先级的补丁会生效。如果多个补丁的优先级一致，列表里靠前的补丁生效。
-* `Exclusive`（独占）：如果某一个补丁使用此优先级，那其他所有补丁都无效。如果多个补丁使用此优先级， **所有补丁都不生效** ，并且显示错误信息。
+* `Low`、`Medium` 或 `High`：只有最高优先级的补丁会被应用。如果多个补丁的优先级一致，则按加载顺序 + 补丁顺序列表中靠前的补丁生效。
+* `Exclusive`：如果某一个补丁使用此优先级，那么其他所有补丁都无效。如果多个补丁使用此优先级，**所有补丁都不生效**，并且显示错误信息。
```
```suggestion
* `Low`（低）、`Medium`（中）或 `High`（高）：只有最高优先级的补丁会被应用。如果多个补丁的优先级一致，则按加载顺序 + 补丁顺序列表中靠前的补丁生效。
* `Exclusive`（独占）：如果某一个补丁使用此优先级，那么其他所有补丁都无效。如果多个补丁使用此优先级，**所有补丁都不生效**，并且显示错误信息。
```

SummerFleur2997 reviewed (commented):
slightly changes

SummerFleur2997 committed:
Apply changes from code review

SummerFleur2997 committed:
fix inappropriate expression

SummerFleur2997 committed:
fix inappropriate expression
- 帧 -> 刻
- 替换 -> 修改（in action-editimage.md）
- and other sentences

SummerFleur2997 commented on ContentPatcher/docs/zh/author-guide/action-editmap.md:
```diff
@@ -269,7 +269,7 @@ Content Patcher 会按以下方法处理 `FromFile` 地图内引用的图块集
 `TextOperations` 字段可以编辑一个现有地图属性的值（请参阅[文本操作](../author-guide.md#text-operations)）。
 
-有效的地图属性格式只有 `["MapProperties", "PropertyName"]`，其中 PropertyName 是要更改的地图属性名称。
+`TextOperations` 在此处的 `Target` 路径只允许 `["MapProperties", "PropertyName"]`，其中 PropertyName 是要更改的地图属性名称。
```
已修改，不确定新表述是否符合实际情况。

Fixed but not sure if the new expression match the reality.

此处附上英文原文 En doc original text attached below:
> The only valid path format is `["MapProperties", "PropertyName"]` where `PropertyName` is the
> name of the map property to change.

Mushymato replied:
的确是这个意思，looks good to me

SummerFleur2997 commented on ContentPatcher/docs/zh/conditions-api.md:
```diff
@@ -191,13 +191,13 @@ var conditions = api.ParseConditions(
 
 
-条件 API 需要在 `GameLaunched` 事件发生后两帧（Tick）起才可使用。这和 Content Patcher 的生命周期有关：
+条件 API 需要在 `GameLaunched` 事件发生后两刻（Tick）起才可使用。这和 Content Patcher 的生命周期有关：
```
此处改为刻（参考 Minecraft 的翻译）

refer to the relevant terms in minecraft, use `刻` instead of `帧`

Mushymato replied:
fine with this word choice, can note in translation-notes.md for future.

SummerFleur2997 reviewed (commented):
不确定的更改，等待复核。

Uncertain changes, pending review.

SummerFleur2997 committed:
Describe "Tick" on translation-notes

Pathoschild committed:
avoid specific editor name when linking to a wiki anyone can edit

Pathoschild committed:
keep links to official wiki

We're discussing the wiki split separately, so I'll keep these links as-is for now. This commit can be reverted later if needed.

Pathoschild committed:
fix typos

Pathoschild commented:
Merged into `develop`; thanks for all the work!
